package.json

```diff
@@ -33,7 +33,10 @@
     "@types/react": "16.9.19",
     "broccoli": "3.3.3",
     "ethereum-transaction-debugger": "https://registry.yarnpkg.com/@favware/skip-dependency/-/skip-dependency-1.0.2.tgz",
-    "scrypt": "https://registry.yarnpkg.com/@favware/skip-dependency/-/skip-dependency-1.0.2.tgz"
+    "scrypt": "https://registry.yarnpkg.com/@favware/skip-dependency/-/skip-dependency-1.0.2.tgz",
+    "**/react": "16.8.0",
+    "**/react-dom": "16.8.0",
+    "@babel/core": "7.8.3"
```

Comment:
Wondering whether these should be in the integration-test package?

Reply:
No, but we might want to move them inside some of the other packages, e.g. `xstate-wallet`. 

Currently if we run `yarn` from the top level, these resolutions will do their job. But if we `yarn add` from an individual package, I think they will not 😟